packages/api/src/error.ts

```diff
@@ -96,3 +97,12 @@ export class UserInputError extends ApolloError {
     super(`User Input Error: \n${actualError}`, ErrorCode.UserInputError)
   }
 }
+
+export class DuplicatePageSlugError extends ApolloError {
```

Comment:
Is this class will be used a lot afterward?

Reply:
For this kind of error